feed item tile cleanup (bug 1134356)

spasovski opened the pull request:
This covers the typography/curve issues but also starts an initial phase of CSS refactoring that still needs more work.

An important change to remember is this image (where the white layer refers to a BG image):

![](https://dl.dropboxusercontent.com/u/545688/mozilla/Screen%20Shot%202015-02-20%20at%201.31.10%20PM.png)

![](http://i.imgur.com/UIeeomB.png)

![](http://i.imgur.com/MtqDRmb.png)

ngokevin commented:
grep for feed-curve, i think i added that for a reason

ngokevin commented on src/media/css/feed/feed.styl:
```diff
@@ -222,35 +174,79 @@
 }
 
+// Promo tile classes.
+.feed-item-info {
```
should we call these feed-promo-tile and move them into maybe a ```promo-tile.styl``` file? These don't apply to all feed items right?

ngokevin commented:
- For Promo Collections with backgrounds, can we move the title down a bit/add padding below the top of the curve? it looks a bit too close to the top of the curve
- A bit too much bottom padding relative to the top on the Collection Promo Background Desc

ngokevin commented:
I think the text can be moved down here too

![screen shot 2015-02-23 at 1 25 38 pm](https://cloud.githubusercontent.com/assets/674727/6337584/82774e7a-bb5f-11e4-9805-0983cdb2b8fb.png)

ngokevin commented on src/media/css/feed/collections.styl:
```diff
@@ -55,7 +55,14 @@
-    width: $mobile-content-width;
+}
+
+.feed-landing-header-mobile .feed-item {
```
We can't do this because the tiles don't stretch

![screen shot 2015-02-23 at 1 26 55 pm](https://cloud.githubusercontent.com/assets/674727/6337619/aa8675e4-bb5f-11e4-9ae8-de2d2b1a1ee0.png)

spasovski commented:
I addressed most of the comments I think. Great find on the landing page resize transitions - we *could* create a new feed-promo-tile but some of these changes are meant to affect the landing pages also and do.

ngokevin commented:
I still see we're trying to stretch the Feed tiles. I think we should leave them how they were at 300px fixed and we can try to stretch them later.

![screen shot 2015-02-26 at 11 28 05 am](https://cloud.githubusercontent.com/assets/674727/6399943/97775c46-bdaa-11e4-8df8-c0f2c15ffa3b.png)

ngokevin commented:
r+ nice
